<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionControl_UI</class>
 <widget class="QWidget" name="ConnectionControl_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBoxConnection">
     <property name="title">
      <string>Подключение</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_TCPServer">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Сервер</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_TCPHostAddress">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_TCPDeviceAvailable">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Доступные устройства</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QComboBox" name="comboBox_TCPDevices">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_RSPortSelect">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Выбор порта</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_RSPorts">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QRadioButton" name="radioButton_RS">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Com-порт</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_TCP">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>TCP</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_RS</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_TCPServer</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_RS</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_TCPHostAddress</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_RS</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_TCPDeviceAvailable</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_RS</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_TCPDevices</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_TCP</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_RSPortSelect</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_TCP</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_RSPorts</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
